builder/builder.go

```diff
@@ -655,14 +656,36 @@ func (b *Builder) buildUpdateWithNewSwupd(timer *stopWatch, mixVersion uint32, m
 	for _, f := range mom.Files {
 		fmt.Printf("- %-20s %d\n", f.Name, f.Version)
 	}
-	if !skipSigning {
+
+	if skipSigning {
+		fmt.Println("SKIPPING manifest signing.")
+	} else {
 		fmt.Println("Signing manifest.")
 		err = b.SignManifestMoM()
 		if err != nil {
 			return err
 		}
+	}
+
+	thisVersionDir := filepath.Join(outputDir, fmt.Sprint(mixVersion))
+	fmt.Println("Compressing Manifest.MoM")
+	momF := filepath.Join(thisVersionDir, "Manifest.MoM")
+	if skipSigning {
+		err = createCompressedArchive(momF+".tar", momF)
 	} else {
-		fmt.Println("SKIPPING manifest signing.")
+		err = createCompressedArchive(momF+".tar", momF, momF+".sig")
```

Comment:
@tmarcu @matthewrsj after writing this, I've noticed that when using Mixer we don't do this. But the Manifest.MoM.tar in Clear Linux do have the sig included. Should I take it off (not include the signature in the tar)?

Reply:
if it is done elsewhere we should consider doing it here to utilize the very helpful `createCompressedArchive`